lib/core/include/irods/procApiRequest.h

```diff
@@ -29,6 +29,26 @@ extern "C" {
 /// \return An integer indicating success or failure.
 /// \retval >=0 On success.
 /// \retval  <0 On failure.
+///
+/// \code{.c}
+/// rcComm_t* conn = // Our iRODS connection.
+///
+/// gridConfigurationInp_t input;
+/// memset(&input, 0, sizeof(gridConfigurationInp_t));
+/// strcpy(input.name_space, "delay_server");
+/// strcpy(input.option_name, "leader");
+///
+/// gridConfigurationOut_t* output = NULL;
+///
+/// int ec = procApiRequest(conn, GET_GRID_CONFIGURATION_VALUE_APN, &input, NULL, (void**) &output, NULL);
+/// if (ec != 0) {
+///     // Handle error.
```

Comment:
Do we need to free the output in the error case, too? Or does it depend on the API?

Reply:
I think it depends on the API.

Reply:
I'll likely change that `!= 0` to `< 0` on the squash.